### ISSUE TITLE
Segment task training from scratch

### ISSUE THREAD

**Issue by QiqLiang** [ultralytics/ultralytics#19198]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hi,I want to use yolov8n-seg model for my segmentation task, but for some reasons I need to replace all silu activation by relu activation. I have two questions,
1. Should I train the model from scratch using coco dataset or should I just use your pretrained model to finetune my model using my own dataset ?
2. If I need to train from scratch, where can I close the auto downloading of yolov11n.pt? Cuz I tried to train from scratch but always need to download your pretrained model and since I can’t surf the net, it always gets stucked.what should I do to solve it?

**Comment by glenn-jocher** [comment]
👋 Hello @QiqLiang, thank you for your interest in Ultralytics 🚀! We appreciate your questions and will do our best to assist. This is an automated response to guide you while an Ultralytics engineer reviews your issue shortly.

### Recommendations

We recommend checking out our [Docs](https://docs.ultralytics.com/) for useful guidance. You'll find detailed explanations, including examples for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) setups. Many common setups and questions regarding training and customization, such as modifying activation functions and adjusting training workflows, may already be covered there.

### Specific Suggestions for Your Issue

1. If this is related to training from scratch or using custom models, we recommend consulting the [Custom Training Guide](https://docs.ultralytics.com/guides/model-training-tips/) for tips on training best practices.
2. If you are facing downloading issues and wish to avoid auto-downloading the pretrained `.pt` weights, you can skip them by providing the `--weights ''` argument in your training command (this can stop the automatic use of pretrained models).

### 🐛 Bug Reports  

For debugging or replication of unusual behaviors, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/). This is essential to help us understand and resolve your issue effectively. If possible, include your code snippet or configuration and any relevant logs.

### Community Support  

💬 Join the discussion in our vibrant Ultralytics community!  
- For real-time interactions, join our [Discord](https://discord.com/invite/ultralytics) 🎧.  
- Get involved in discussions at [Discourse](https://community.ultralytics.com/).  
- Share your journey and learn with others on our [Subreddit](https://reddit.com/r/Ultralytics).  

### Upgrade  

Please ensure that your `ultralytics` package is up-to-date to rule out legacy issues. Update your environment using the command below:  

```bash
pip install -U ultralytics
```

### Verified Environments  

If you're facing environment-specific issues, consider exploring one of our verified environments to ensure all dependencies are pre-configured:
- Free GPU environments: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>  
- [GCP Quickstart](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)  
- [AWS Quickstart](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)  
- [Docker Quickstart](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>  

### Status  

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>  

If the above badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests confirm correct operation across macOS, Windows, and Ubuntu for all [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/).

We hope this helps! An Ultralytics team member will follow up with you soon 😊

**Comment by QiqLiang** [comment]
> 👋 Hello [@QiqLiang](https://github.com/QiqLiang), thank you for your interest in Ultralytics 🚀! We appreciate your questions and will do our best to assist. This is an automated response to guide you while an Ultralytics engineer reviews your issue shortly.
> 
> ### Recommendations
> 
> We recommend checking out our [Docs](https://docs.ultralytics.com/) for useful guidance. You'll find detailed explanations, including examples for [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) setups. Many common setups and questions regarding training and customization, such as modifying activation functions and adjusting training workflows, may already be covered there.
> 
> ### Specific Suggestions for Your Issue
> 
> 1. If this is related to training from scratch or using custom models, we recommend consulting the [Custom Training Guide](https://docs.ultralytics.com/guides/model-training-tips/) for tips on training best practices.
> 2. If you are facing downloading issues and wish to avoid auto-downloading the pretrained `.pt` weights, you can skip them by providing the `--weights ''` argument in your training command (this can stop the automatic use of pretrained models).
> 
> ### 🐛 Bug Reports
> 
> For debugging or replication of unusual behaviors, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/). This is essential to help us understand and resolve your issue effectively. If possible, include your code snippet or configuration and any relevant logs.
> 
> ### Community Support
> 
> 💬 Join the discussion in our vibrant Ultralytics community!
> 
> * For real-time interactions, join our [Discord](https://discord.com/invite/ultralytics) 🎧.
> * Get involved in discussions at [Discourse](https://community.ultralytics.com/).
> * Share your journey and learn with others on our [Subreddit](https://reddit.com/r/Ultralytics).
> 
> ### Upgrade
> 
> Please ensure that your `ultralytics` package is up-to-date to rule out legacy issues. Update your environment using the command below:
> 
> pip install -U ultralytics
> ### Verified Environments
> 
> If you're facing environment-specific issues, consider exploring one of our verified environments to ensure all dependencies are pre-configured:
> 
> * Free GPU environments: [![Run on Gradient](https://camo.githubusercontent.com/6df71b7d7e0b09a2e97776f416bcd40acf48fc87337f8bcce48e4235537daf1f/68747470733a2f2f6173736574732e706170657273706163652e696f2f696d672f6772616469656e742d62616467652e737667)](https://console.paperspace.com/github/ultralytics/ultralytics) [![Open In Colab](https://camo.githubusercontent.com/96889048f8a9014fdeba2a891f97150c6aac6e723f5190236b10215a97ed41f3/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667)](https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb) [![Open In Kaggle](https://camo.githubusercontent.com/c7135949c5c6882489e68f4af05a78a759460a4db256b86df3097e04419b4d9e/68747470733a2f2f6b6167676c652e636f6d2f7374617469632f696d616765732f6f70656e2d696e2d6b6167676c652e737667)](https://www.kaggle.com/models/ultralytics/yolo11)
> * [GCP Quickstart](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
> * [AWS Quickstart](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
> * [Docker Quickstart](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) [![Docker Pulls](https://camo.githubusercontent.com/6dcbe0a245fadcfd5202d971a92467bd2fcab86efebb2228d73468152c680573/68747470733a2f2f696d672e736869656c64732e696f2f646f636b65722f70756c6c732f756c7472616c79746963732f756c7472616c79746963733f6c6f676f3d646f636b6572)](https://hub.docker.com/r/ultralytics/ultralytics)
> 
> ### Status
> 
> [![Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg)](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule)
> 
> If the above badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are passing. These tests confirm correct operation across macOS, Windows, and Ubuntu for all [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/).
> 
> We hope this helps! An Ultralytics team member will follow up with you soon 😊

But I got error if I set weights=‘’ in model.train()? ‘weights’ is not a valid YOLO argument.

**Comment by QiqLiang** [comment]
It still tries to download even if I have set the pretrained parameter to false.

**Comment by Y-T-G** [comment]
1. You can train from scratch if you want.
2. It's downloaded for AMP checks. You can keep a copy in the local directory and it won't download it and use the local one.
